lib/modules/dosomething/dosomething_helpers/dosomething_helpers.module

```diff
@@ -576,8 +576,8 @@ function dosomething_helpers_node_metadata_wrapper($node, $ignore_language = FAL
  *   ISO-code of current language
  */
 function dosomething_helpers_get_current_language_code() {
-  global $language;
-  return $language->language;
+  global $language_content;
```

Comment:
it doesn't look like it's used for anything but content, but maybe rename to `get_current_node_lang_code`